src/components/articleCardTemplate.tsx

```diff
@@ -0,0 +1,78 @@
+import Image from 'next/image';
+import { ComponentProps } from 'react';
+import {
+  Card as Cd,
+  CardContent,
+  CardDescription,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from '@/components/ui/card';
+import {
+  Avatar,
+  AvatarFallback,
+  AvatarImage,
+} from '@/components/ui/avatar';
+import { PartyPopper, ThumbsUp } from 'lucide-react';
+import SocialLink from './custom/SocialLink';
+
+type ArticleCardTitleProps = {
+  title: '熱門文章' | '推薦文章';

```

Comment:
我的話可能會另外用 type 去控制樣式，type 可以另外用 enum
```suggestion
  title: '熱門文章' | '推薦文章';
  type:  'hot' | 'recommend';

```